### DISCUSSION TITLE
lv_cpython embedded linux installation

### DISCUSSION THREAD

**Discussion by favnec5** [kdschlosser/lv_cpython#12] [General]
Hello,
nice job !
Imagining a linux embedded Arm device with python, 
is it possible to install lv_cpython manually on the device (without git) ?
It should also be possible to activate full screen mode.
regards,
Thomas

**Comment by kdschlosser** [comment]
yeah you have to download the repository as a zip file and then unzip it onto whatever device you are going to be running it on. You need to also install a compatible C compiler and linker on that device and then use pip to install it. This will compile and install the library into the python you have running on that device.

There is no way to compile on say a desktop and then transfer the compiled files to the embedded device because the Linux kernels are going to be different and that would cause an issue. When dealing with Linux It MUST be compiled on the device it is going to be running on. This is because of build requirements like SDL being used.

**Comment by favnec5** [comment]
Ok many thanks !
i'll go to test it.
wishing you a speedy recovery
regards,
Thomas

**Reply by bhupiister** [reply]
@favnec5 @kdschlosser How can we actually build it on arm device? i am trying to build this for luckfox pico ultra w, which is am arm based device.
